v2/internal/controllers/reconcile_policy_test.go

```diff
@@ -198,7 +250,7 @@ func Test_ReconcilePolicySkip_CreatesSecretAndConfigMap(t *testing.T) {
 	tc.Patch(old, acct)
 }
 
-func testDeleteSkipped(t *testing.T, policy string) {
+func testDeleteSkipped(t *testing.T, policy string, onNamespace bool) {
```

Comment:
Should this actually two methods that possibly share some helper methods?

If not, consider using a constant - `testDeleteSkipped(t, "skip, false)` is pretty opaque as to what the **false** (or **true**) means.